sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -24,3 +24,25 @@ select left("abcd", 2), left("abcd", 5), left("abcd", '2'), left("abcd", null);
 select left(null, -2), left("abcd", -2), left("abcd", 0), left("abcd", 'a');
 select right("abcd", 2), right("abcd", 5), right("abcd", '2'), right("abcd", null);
 select right(null, -2), right("abcd", -2), right("abcd", 0), right("abcd", 'a');
+
+-- Concatenate binary inputs
+SELECT (col1 || col2 || col3 || col4) col
```

Comment:
Let us add a new suite in `.../sql/core/src/test/resources/sql-tests/inputs/typeCoercion/native/`

Reply:
ok